Website/src/rendering/src/components/SpeakerInformation.tsx

```diff
@@ -5,54 +5,63 @@ import {
   faInstagram,
 } from '@fortawesome/free-brands-svg-icons';
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { ComponentProps } from 'lib/component-props';
+import { Field, ImageField, Image, RichText } from '@sitecore-jss/sitecore-jss-nextjs';
 
-const SpeakerInformation = (): JSX.Element => (
+type SpeakerInformationProps = ComponentProps & {
+  fields: {
+    Name: Field<string>;
+    Role: Field<string>;
+    Image: ImageField;
+    Position: Field<string>;
+    Company: Field<string>;
+    Country: Field<string>;
+    Description: Field<string>;
+    Facebook: Field<string>;
+    Twitter: Field<string>;
+    Linkedin: Field<string>;
+  };
+};
+
+const SpeakerInformation = (props: SpeakerInformationProps): JSX.Element => (
   <section className="section">
     <div className="section__content left__content">
       <div className="grid grid-cols-1 md:grid-cols-4 gap-10">
         <div className="col-span-1 md:col-span-1">
-          <img src="/data/media/img/speakers/mary-asada.jpeg" alt="Mary Asada" />
+          <Image field={props.fields?.Image} alt={props.fields?.Name.value} />
           <div>
-            <FontAwesomeIcon className="icon h-4 m-2 inline text-blue" icon={faFacebookF} />
-            <FontAwesomeIcon className="icon h-4 m-2 inline text-blue" icon={faTwitter} />
-            <FontAwesomeIcon className="icon h-4 m-2 inline text-blue" icon={faLinkedinIn} />
-            <FontAwesomeIcon className="icon h-4 m-2 inline text-blue" icon={faInstagram} />
+            {/* TODO: To be turned into links */}
+            <a href={props.fields.Facebook.value}>
```

Comment:
For each social media field, we should first check whether the field has a value before rendering the icon associated with it.